.eslintignore

```diff
@@ -1,4 +1,4 @@
-docs/_build/
-clients/web/dev/
+build/
 clients/web/static/built/
 clients/web/test/lib/
+docs/_build/
```

Comment:
Could you add `plugins/*/web_client/extra/`?  Many plugins use [this directory](https://github.com/girder/large_image/blob/master/.eslintignore) to store vendored dependencies.  Given the limitation on ignore files (mentioned in my other comment), this will at least allow running `eslint` standalone to work in *most* cases.

Reply:
Sure.